manager.go

```diff
@@ -0,0 +1,53 @@
+package inflight
+
+import (
+	fq "github.com/aaron-prindle/fq-apiserver"
+	"k8s.io/apimachinery/pkg/util/clock"
+)
+
+type sharedDispatcher struct {
+	producers map[PriorityBand]*Dispatcher
+}
+
+func queuesForPriority(priority PriorityBand, queues []*Queue) []*Queue {
+	// TODO(aaron-prindle) change this to actual impl
+	return []*Queue{queues[priority]}
+}
+
+func newSharedDispatcher(queues []*Queue) *sharedDispatcher {
+
+	mgr := &sharedDispatcher{
+		producers: make(map[PriorityBand]*Dispatcher),
+	}
+
+	clock := clock.RealClock{}
+	for _, priority := range Priorities {
+
+		queuesforpriority := queuesForPriority(priority, queues)
+		fqqueues := make([]fq.FQQueue, len(queuesforpriority), len(queuesforpriority))
+		for i := range queuesforpriority {
+			fqqueues[i] = queuesforpriority[i]
+		}
+		mgr.producers[priority] = &Dispatcher{
+			queues: queues,
+			ACV:    1,
+			// ACV:         ACV(priority, queues),
+			fqScheduler: NewFQScheduler(fqqueues, clock),
+		}
+	}
+	// TODO(aaron-prindle) FIX - this eventually needs to be dynamic...
```

Comment:
Yes, and the design needs to support concurrent goroutines that respond to configuration object changes.